packages/web/src/components/nav-banner/NavBanner.tsx

```diff
@@ -1,13 +1,18 @@
 import { ReactElement } from 'react'
 
-import { removeNullable } from '@audius/common/utils'
 import { Flex, Button, PopupMenu, IconSort as SortIcon } from '@audius/harmony'
 import cn from 'classnames'
 
 import { ClientOnly } from 'components/client-only/ClientOnly'
 
 import styles from './NavBanner.module.css'
 
+function removeNullable<T>(
```

Comment:
Could you pull this from `@audius/common/utils/typeUtils` instead?

Reply:
No it complains that it's not exported when you try to import from a deeper route

Reply:
Ah ok, because of this I think: https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/tsconfig.json#L27